### ISSUE TITLE
Config Rows Definition - remove deprecated properties.

### ISSUE THREAD

**Issue by pivnicek** [keboola/db-extractor-mysql#57]
The enable/disable for configRow configurations is handled via the docker runner api so the config property is deprecated.

Properties to be removed for ConfigRow definition:
- id
- name
- enabled

This may require a minor change to the frontend to filter out these properties before saving.

**Comment by Halama** [comment]
what about that `id` is it anyhow useful? I think it can be removed.

![image](https://user-images.githubusercontent.com/903531/44900338-a2f15700-ad04-11e8-8505-b0fc52a02ece.png)

**Comment by pivnicek** [comment]
yes, let's get rid of that at the same time, good idea.  It is redundant with the rowConfigId

**Comment by Halama** [comment]
do you know if ui is using it? If not we can remove it from teradata rightnow, because it is not used inside the extractor.

**Comment by pivnicek** [comment]
I'm 95% sure that it is not used, but I'll confirm right now, brb

**Comment by pivnicek** [comment]
Yes, I think we are safe to remove `id` from the configRow definition.  Btw, similarly, I think we can remove `name` too as it uses the configRow name.
